ckanext/dcatapit/harvesters/csw_harvester.py

```diff
@@ -13,6 +15,25 @@
 
 log = logging.getLogger(__name__)
 
+class ISOTextGroup(ISOElement):
```

Comment:
You are using
**conformity-specification-title** for both identifier and title as we discussed.
So this piece of code can be removed